photo_points/wwwroot/css/photo-card.css

```diff
@@ -1,4 +1,5 @@
 .card-back {
```

Comment:
This css is for a card view that no longer is in the project.. May add that view again if I have time.